src/monopoly/view/monopoly_view.js

```diff
@@ -238,7 +238,51 @@ MonopolyView.prototype.drawCard = function(xPos, yPos, width, height, text, colo
     card.addChild(cardText);
 
     return card;
-}
+};
+
+
+
+MonopolyView.prototype.setupPropertyClick = function(tile, property){
+    tile.interactive = true;
+    var context = this;
+    //this sets the clickable area - doesnt work
+    //tile.hitArea = new PIXI.Rectangle(0, 0, constants.tileShortSide, constants.tileLongSide);
+    
+    tile.click = function(mouseData){
+        if(context.game.state == PROPOSE_TRADE){
+           // console.log("CLICK PROP!" + property.name);
+            if(context.game.trade == null){
+                context.game.createTrade();
+            }else{
+                context.game.addPropertyToTrade(property);
+                // console.log(context.game.trade.answering_player.name);
+            }
```

Comment:
could do a nifty shortcut here: 

    !context.game.trade ? context.game.createTrade() : context.game.addPropertyToTrade(property);

also it's generally better style to do if(!x) rather than if(x==null)